### DISCUSSION TITLE
Name-prefixed Route Groups in Laravel?

### DISCUSSION THREAD

**Discussion by cjholowatyj** [tighten/ziggy#453] [Q&A]
So I recently decided to refactor my code to make the route names in my app cleaner. I decided to use Laravel's Route Group Name Prefixing (https://laravel.com/docs/8.x/routing#route-group-name-prefixes) but either me or Ziggy is missing something when it comes to using Ziggy to pass routes to inertia/vue...
-----
Sample Laravel Route:
```
Route::middleware(['web'])->name('jetstream.')->group(function(){
	Route::get('/workspaces/{workspace}',[WorkspaceController::class,'show'])->name('workspaces.show');
});
```
Console Error:
```
Uncaught (in promise) Error: Ziggy error: route 'jetstream.worskspaces.show' is not in the route list.
```

Any suggestions for a Laravel novice to get Ziggy to recognize this route properly? Thanks in advance.

**Comment by bakerkretzmar** [comment]
In that particular console error it looks like there's a typo in "workspaces," could that be it?
